server/cmd/mmctl/commands/user.go

```diff
@@ -799,6 +800,10 @@ func searchUserCmdF(c client.Client, cmd *cobra.Command, args []string) error {
 			User:        user,
 			Deactivated: !(user.DeleteAt == 0),
 		}
+		if user.AuthData != nil {
```

Comment:
For email/password, this will just output an empty string. I wonder if we should make the output more explicit instead.

Reply:
Empty seems fine.